client/src/js/app.js

```diff
@@ -203,6 +203,9 @@ function httpConfig($httpProvider) {
 
   // register error handling interceptor
   $httpProvider.interceptors.push('ErrorInterceptor');
+
+  // perf - applyAsync
+  $httpProvider.useApplyAsync(true);
```

Comment:
It's not clear if this will help or not, but I can imagine three places where this will help:
  1. Typeaheads/validators (such as the hospital_no) will trigger less digest calls.  We use these quite a bit in our application, shipping lots of small requests rapidly over a local network.
  2. Modules were we add new grid rows to Stock/Vouchers/Transactions that prompt HTTP requests.  For example, when we add voucher tools, those often trigger several HTTP requests from the Voucher module.  This will stack those requests to make the browser do fewer $digest calls.
  3. Each ui-grid cellTemplate seems to send an HTTP request.  I'm not sure why - maybe we need to debug this somewhere, but for the moment, this might help.

Again, this might be a bad idea, but we have to try it to find out... from the documentation, it doesn't look like it will _hurt_ anything, just might not help as much as we hope.